### DISCUSSION TITLE
Unexpected settings change?

### DISCUSSION THREAD

**Discussion by TripleFJeff** [gopro/labs#1135] [Q&A]
I have several Hero 8s that I have set up to be fully automatic cameras for race cars. They wake on power (they are wired into the car) and start based on GPS speed. I had one that unexpectedly started while not moving, and it seems it was in time lapse mode. If I look at the metadata from the files before and after this one, which worked as normal, there are differences in all 3. Any idea what may have happened? Is there a reference for how to interpret the metadata? File 356 is the one that started while not moving and is time lapse.

![image](https://github.com/user-attachments/assets/b87978da-abef-4c0b-8607-16d8a07e5a3e)

**Comment by dnewman-gpsw** [comment]
The metadata is got going tell you what happen, other than it recorded timelapse. CMOD is camera mode, these are internal use code, likely used by Quik.  You boot script could request video mode before starting a GPS speed capture.  Otherwise is could be a random event.
